### DISCUSSION TITLE
【灌水】10M上限？不存在的！

### DISCUSSION THREAD

**Discussion by capacitor1** [share121/inter-knot#652] [其他]
单个文件发不了 我分卷总行吧doge

提示：由于GitHub卡文件后缀，所以下载完分卷之后，需要打开cmd并执行下列命令：
```copy /b *.zip Original.zip```
意为：将所有后缀为zip的文件合并成Original.zip。
之后，只需要解压Original.zip就能得到所需的文件了。

[草白露_1007741843_AVC_[BV1Ts4y1a7CG].zip.001.zip](https://github.com/user-attachments/files/16568560/_1007741843_AVC_.BV1Ts4y1a7CG.zip.001.zip)
[草白露_1007741843_AVC_[BV1Ts4y1a7CG].zip.002.zip](https://github.com/user-attachments/files/16568561/_1007741843_AVC_.BV1Ts4y1a7CG.zip.002.zip)
[草白露_1007741843_AVC_[BV1Ts4y1a7CG].zip.003.zip](https://github.com/user-attachments/files/16568563/_1007741843_AVC_.BV1Ts4y1a7CG.zip.003.zip)
[草白露_1007741843_AVC_[BV1Ts4y1a7CG].zip.004.zip](https://github.com/user-attachments/files/16568564/_1007741843_AVC_.BV1Ts4y1a7CG.zip.004.zip)
[草白露_1007741843_AVC_[BV1Ts4y1a7CG].zip.005.zip](https://github.com/user-attachments/files/16568565/_1007741843_AVC_.BV1Ts4y1a7CG.zip.005.zip)
[草白露_1007741843_AVC_[BV1Ts4y1a7CG].zip.006.zip](https://github.com/user-attachments/files/16568566/_1007741843_AVC_.BV1Ts4y1a7CG.zip.006.zip)
[草白露_1007741843_AVC_[BV1Ts4y1a7CG].zip.007.zip](https://github.com/user-attachments/files/16568567/_1007741843_AVC_.BV1Ts4y1a7CG.zip.007.zip)
[草白露_1007741843_AVC_[BV1Ts4y1a7CG].zip.008.zip](https://github.com/user-attachments/files/16568568/_1007741843_AVC_.BV1Ts4y1a7CG.zip.008.zip)
[草白露_1007741843_AVC_[BV1Ts4y1a7CG].zip.009.zip](https://github.com/user-attachments/files/16568570/_1007741843_AVC_.BV1Ts4y1a7CG.zip.009.zip)
[草白露_1007741843_AVC_[BV1Ts4y1a7CG].zip.010.zip](https://github.com/user-attachments/files/16568571/_1007741843_AVC_.BV1Ts4y1a7CG.zip.010.zip)
[草白露_1007741843_AVC_[BV1Ts4y1a7CG].zip.011.zip](https://github.com/user-attachments/files/16568572/_1007741843_AVC_.BV1Ts4y1a7CG.zip.011.zip)
[草白露_1007741843_AVC_[BV1Ts4y1a7CG].zip.012.zip](https://github.com/user-attachments/files/16568573/_1007741843_AVC_.BV1Ts4y1a7CG.zip.012.zip)
[草白露_1007741843_AVC_[BV1Ts4y1a7CG].zip.013.zip](https://github.com/user-attachments/files/16568574/_1007741843_AVC_.BV1Ts4y1a7CG.zip.013.zip)
[草白露_1007741843_AVC_[BV1Ts4y1a7CG].zip.014.zip](https://github.com/user-attachments/files/16568575/_1007741843_AVC_.BV1Ts4y1a7CG.zip.014.zip)
[草白露_1007741843_AVC_[BV1Ts4y1a7CG].zip.015.zip](https://github.com/user-attachments/files/16568576/_1007741843_AVC_.BV1Ts4y1a7CG.zip.015.zip)
[草白露_1007741843_AVC_[BV1Ts4y1a7CG].zip.016.zip](https://github.com/user-attachments/files/16568578/_1007741843_AVC_.BV1Ts4y1a7CG.zip.016.zip)
[草白露_1007741843_AVC_[BV1Ts4y1a7CG].zip.017.zip](https://github.com/user-attachments/files/16568579/_1007741843_AVC_.BV1Ts4y1a7CG.zip.017.zip)
[草白露_1007741843_AVC_[BV1Ts4y1a7CG].zip.018.zip](https://github.com/user-attachments/files/16568580/_1007741843_AVC_.BV1Ts4y1a7CG.zip.018.zip)

**Comment by Yizakl** [comment]
布什·戈门

**Comment by lllliufeng** [comment]
还有高手？

**Comment by ZhuYuan-148** [comment]
这这这…这是什么奇怪的东西，但姑且压缩包里的内容倒是了解了…

**Comment by Yizakl** [comment]
看到文件名前三个字就明白了一切

**Comment by lizard315315** [comment]
我倒要看看这到底是什么🤪

**Reply by lizard315315** [reply]
抽象

**Comment by capacitor1** [comment]
> Reply::这这这…这是什么奇怪的东西，但姑且压缩包里的内容倒是了解了…

朱鸢姐姐千万不要误会！！

通过文件名中的BV1Ts4y1a7CG可知，这就是个正常的B站视频！
https://www.bilibili.com/video/BV1Ts4y1a7CG/
